setup.cfg

```
@@ -32,7 +32,7 @@ install_requires =
     nest-asyncio
     requests
     pins
-    rsconnect-python
+    rsconnect-python>=1.12.1
```

```suggestion
    rsconnect-python>=1.11.0
```

I am able to deploy with 1.11.0, let's relax this a little bit. LMK if you are encountering different issues!

---

I just left another comment in #131. I am thinking maybe it should be pinned to:

```
rsconnect-python>=1.8.0
```

---

While we're pinning versions, there was a breaking bug in 1.10, so I think the official list is something like:
`rsconnect-python>=1.8.0, <1.10, >=1.11`, if I have all my < and > in the right place 😄